<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>601</width>
      <height>931</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="StateWidget" name="widget_1" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>海雀1号</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="StateWidget" name="widget_5" native="true">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>鲲鹏2号</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="StateWidget" name="widget_2" native="true">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>海雀2号</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="StateWidget" name="widget_4" native="true">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>鲲鹏1号</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="StateWidget" name="widget_6" native="true">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>鲲鹏3号</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="StateWidget" name="widget_3" native="true">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>海雀3号</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="StatusWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>20</y>
      <width>581</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="WarnWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>560</y>
      <width>581</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>城鹰5G巡检安防平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>北京时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>在线设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>离线设备</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="widget_map" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>691</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>200</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>200</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StateWidget</class>
   <extends>QWidget</extends>
   <header location="global">statewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusWidget</class>
   <extends>QWidget</extends>
   <header>statuswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WarnWidget</class>
   <extends>QWidget</extends>
   <header>warnwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
